BE/src/main/java/kr/codesquad/issuetracker/service/MemberService.java

```diff
@@ -52,4 +53,13 @@ private MemberResponseDto join(AccessTokenResponseDto token, UserProfile userInf
 	private boolean isNotMember(String email) {
 		return memberRepository.findByEmail(email).isEmpty();
 	}
+
+	public Member findUserById(Long memberId) {
+		return memberRepository.findById(memberId)
+			.orElseThrow(() -> new CustomException(MEMBER_NOT_FOUND));
+	}
```

Comment:
메소드가 throw나 null을 반환하는 경우 메소드 이름에 throw or null을 선언해주면 사용하는 측에서 예측하기 쉬워집니다.
```suggestion
	public Member findUserByIdOrThrow(Long memberId) {
		return memberRepository.findById(memberId)
			.orElseThrow(() -> new CustomException(MEMBER_NOT_FOUND));
	}
```

Reply:
메서드 이름도 조금더 명시적으로 만들수 있군요 감사합니다 :)